views/layout/header.html.php

```diff
@@ -18,6 +18,7 @@
   <link rel="mask-icon" href="/images/icons/safari-pinned-tab.svg" color="#5bbad5">
   <meta name="msapplication-TileColor" content="#da532c">
   <meta name="theme-color" content="#ffffff">
+  <meta name="client-is-author" content="<?= $personInfo['loggedIn'] && !empty($sketch['person_id']) && $personInfo['details']['person_id'] == $sketch['person_id']? "true" : "false" ?>">
```

Comment:
Since we already have this boolean statement in sketch-editor.html.php as data-read-only, and we grab this value in editor.js, we don't need to update this file at all.

The less code we have the less bugs we can have! It's better to reduce the amount of code where possible.